examples/DAZLS/better_dazls.ipynb

```diff
@@ -0,0 +1,819 @@
+{
```

Comment:
The repo currently has 4 examples of functionality within OpenSTEF. To me it would make sense to make this notebook a 5th example, e.g: 05. Split net load into Components
Could you rename and move the notebook?